risc0/zkvm/src/host/protos/api.proto

```diff
@@ -239,7 +239,7 @@ message TraceEvent {
 
   message MemorySet {
     uint32 addr = 1;
-    uint32 value = 2;
+    bytes region = 2;
```

Comment:
This is a breaking change to protobuf. We can make breaking changes to the API, but we can't make breaking changes to protobuf messages. If we're changing the data type, we need to say:
```suggestion
    uint32 value = 2;
    bytes region = 3;
```